content/md/first-steps/manage-families.md

```diff
@@ -1,112 +1,102 @@
 ---
 id: manage-your-families
 themes: catalog-settings
-title: Manage your **families**
+title: Manage your **families** and **variant families**
```

Comment:
Sorry, you cannot put several part of the title in bold. It's not working currently.